test/transcoding/OpenCL/convert_functions.ll

```diff
@@ -55,13 +57,17 @@ entry:
 ; CHECK-LLVM: call spir_func void @_Z18convert_float_func(float %[[Call]])
 ; CHECK-LLVM: call spir_func void @_Z20convert_uint_satfunc(i32 %[[#]])
 ; CHECK-LLVM: call spir_func void @_Z21convert_float_rtzfunc(float %[[Call]])
+  call spir_func signext i8 @_Z16convert_char_rtei(i32 noundef %0) #1
```

Comment:
Is `CHECK-LLVM` missing here?

Reply:
I think one could be added for that, but I added a check to make sure the convert ends up in the SPIR-V, so I think that should be good enough?

Reply:
Just to be sure, I'd prefer to have a check for reverse translation too, as it's already done for other convert functions in the test

Reply:
oh right, yeah that makes sense. I wasn't considering reverse translation here. I can open a new PR soon with that added.

Reply:
https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/3128